### ISSUE TITLE
Error getting ADB bugreport on API 21

### ISSUE THREAD

**Issue by greenEkatherine** [dotnet/xharness#644]
After app crashed XHarness failed to get bugreport with the error message:
```
fail: Error getting ADB bugreport: Exit code: 255 Standard Output: Standard Error: Failed to get bugreportz version: 'bugreportz -v' returned '/system/bin/sh: bugreportz: not found' (code 0). If the device does not run Android 7.0 or above, try 'adb bugreport' instead. 
```

Build https://dev.azure.com/xamarin/public/_build/results?buildId=41590&view=logs&j=dd18a2d5-2e4c-5ccd-8c00-4b4d347e66ba&t=883bb964-b431-5e95-fb8b-cec2c28425c6

/cc @mattleibow

**Comment by greenEkatherine** [comment]
On old API (<24) we should use `adb bugreport > file.txt` instead of newer `adb bugreport file.zip`
I successfully run it locally with adb and now need to fix inside XHarness.

**Comment by greenEkatherine** [comment]
@mattleibow fix was published in version `1.0.0-prerelease.21378.1`

**Comment by mattleibow** [comment]
Weeee! Updates here I come! Thanks a bunch!
